README.md

```diff
@@ -1,5 +1,55 @@
 # mobile-android-cri-orchestrator
 
+## About
+
+The Credential Issuer (CRI) Orchestrator coordinates identity proofing capability for the [GOV.UK One Login app](https://github.com/govuk-one-login/mobile-android-one-login-app).
```

Comment:
I think the app as a whole will be branded GOV.UK One Login because we won't be able to launch two apps with the same name on the Play Store, but the ID Check journey _within it_ is referred to ID Check v2.